plugins/CoreAdminHome/templates/trackingCodeGenerator.twig

```diff
@@ -20,7 +20,7 @@
             {{ 'General_GoTo2'|translate }}:
             <a href="#/javaScriptTracking">{{ 'CoreAdminHome_JavaScriptTracking'|translate  }}</a>
             <a href="#/imageTracking">{{ 'CoreAdminHome_ImageTracking'|translate }}</a>
-            <a href="#/importServerLogs">{{ 'CoreAdminHome_ImportingServerLogs'|translate }}</a>
```

Comment:
The translation key `CoreAdminHome_ImportingServerLogs` now seems no longer to be used, so we can remove it from the (english) translation file.

Reply:
done